backend/src/controllers/cards/ordenation-card.js

```diff
@@ -0,0 +1,51 @@
+import { connection as knex } from '../../database/connection.js';
+import { refreshUpdateDateBoard } from '../../utils/refresh-update-date-board.js';
+
+const ordenationCards = async (req, res) => {
+	const { cardIdSourcePosition, cardIdDestinationPosition, cardId } = req.body;
+	const limitEditIncrement = cardIdSourcePosition - 1;
+	try{
+
+		await knex.transaction(async trx => {
```

Comment:
Confesso que não entendi pq tem essa primeira verificação, avisando que a nova posição não pode ser igual à última... Pq pelo que entendi, essa rota só vai ser chamada quando uma task ou card for movida.
Este aviso é útil pra caso a pessoa mova o card na tela, mas acabe deixando o card no mesmo lugar... mas aí não se mudaria o número da posição do card, então acho que essa rota não seria chamada. E se fosse chamada, mandar este aviso seria meio desnecessário pq não vai aparecer nada pro cliente no front, aí acho que não precisa de verificação pra quando a posição é igual, só não faz nada mesmo, deixa os ifs só pra quando muda posição. Faz sentido ou estou viajando?

Reply:
Faz sentido sim, eu fiz essa verificação apena pra não chamar o banco de dados caso o usuário mova a task para o mesmo local, más essa verificação pode ser feita no front... Então ela é uma proteção a mais, porem não é muito necessária, vou remover essa verificação e qualquer coisa se necessário volto com ela